### DISCUSSION TITLE
(libreoffice-still) wrong package version

### DISCUSSION THREAD

**Discussion by mclose90-cmd** [chocolatey-community/chocolatey-packages#1937] [Q&A] [answered]
Why did the libreoffice-still package update to version 7.3.4? It's the same version as libreoffice-fresh now. libreoffice-still should be on 7.2.7

**Comment by AdmiringWorm** [comment, marked as answer]
The package uses the update endpoint that LibreOffice itself uses, I would say something may have changed though to make it report the wrong version.

@wget is this something you are able to look into?

**Reply by mclose90-cmd** [reply]
So, I just tested it again and I can confirm that using chocolatey to install libreoffice-still installs version 7.3.4 which is the incorrect version. Thanks for responding.

**Reply by RedBaron2** [reply]
It appears that the update script that libre office uses is putting out only the 7.3.4 version. It appears to be a bug in their website. This is not a problem with the package updater.

**Reply by mclose90-cmd** [reply]
Bug report:
https://bugs.documentfoundation.org/show_bug.cgi?id=149910

**Comment by RedBaron2** [comment]
Since this is due to 7.2.7 being EOL June 12, 2022
I would call this closed. It might be mentioned that the release notes should be checked when this issue arises in the future.

**Reply by mclose90-cmd** [reply]
Thanks.
